quartz-jobs/pom.xml

```diff
@@ -73,14 +73,19 @@
       <version>2.7</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.glassfish.corba</groupId>
+      <artifactId>rmic</artifactId>
+      <version>4.2.4</version>
```

Comment:
Shouldn't this be `<scope>provided</scope>` so its not enforced as a transitive compile time dependency?  The rest of the JARS in this pom are all provided or test.

Reply:
Yes it should, updated now